server/chart_config/climate.json

```diff
@@ -54,5 +54,18 @@
     ],
     "unit": "celsius",
     "isOverview": false
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Mean_concentration_PM2.5",
+    "title": "Mean concentration of PM 2.5",
+    "description": "Mean concentration of PM 2.5",
```

Comment:
please be more descriptive for this.. maybe add that it's for air quality, and measures concentrations of fine inhalable particles, with diameters that are generally 2.5 micrometers and smaller in the air. you could add this link too: https://www.epa.gov/air-trends/particulate-matter-pm25-trends 

this doesn't get displayed -- and goes straight to the translators

Reply:
Done